kiwix.yml

```diff
@@ -55,11 +51,10 @@ all:
     version: "2016-05-16"
     size: "2046925193"
     url: "http://download.kiwix.org/portable/wiktionary/kiwix-0.9+wiktionary_fr_all_2016-05.zip"
-    name: "Wiktionnaire -- Définition, traduction, prononciation, anagramme et synonyme sur le dictionnaire libre Wiktionnaire."
+    name: Wiktionnaire
     language: "fra"
-    description: "Définition, traduction, prononciation, anagramme et synonyme sur le dictionnaire libre Wiktionnaire."
+    description: Définition, traduction, prononciation, anagramme et synonyme sur le dictionnaire libre Wiktionnaire.
```

Comment:
Why not « Le dictionnaire libre » to make it more in line with other languages?

Not only that, but iirc the description is used for cards, where we'd prefer shorter strings?

Maybe we need a short and a long description?